src/script/vco-ie/main_warhammer/wh3_main_ksl_the_great_orthodoxy/missions.lua

```diff
@@ -118,7 +118,7 @@ local missions = {
 					faction wh3_main_ksl_the_great_orthodoxy;
 					total 6;
 				
-					building_level wh3_main_ksl_city_temple_2;

```

Comment:
I would not limit this to Tier 4 buildings because then you will need six capitals with a church, and that can be a pain in the ass for the player. So Tier 3. In order to not penalize the users after an updgrede, change the key `CONSTRUCT_N_OF_A_BUILDING` and use `CONSTRUCT_N_BUILDINGS_FROM` instead. Then use `building_level wh3_main_ksl_city_temple_3` and `wh3_main_ksl_city_temple_4`.